<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cell_expenditure</class>
 <widget class="QWidget" name="cell_expenditure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_outOfRemarks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>223</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>支出类型</string>
      </property>
      <property name="buddy">
       <cstring>cbx_type</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cbx_type">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>餐饮</string>
      </property>
      <item>
       <property name="text">
        <string>餐饮</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>购物</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>日用</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>交通</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>娱乐</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>水果</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>蔬菜</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>运动</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>教育</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>其他</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>支出金额</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="dsbx_account">
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>支出时间</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateTimeEdit" name="dedit_time">
      <property name="displayFormat">
       <string>yyyy/M/d </string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_remarks">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>406</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>备   注</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPlainTextEdit" name="edit_remarks">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>398</width>
      <height>108</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbtn_confirm">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>确  定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_cancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>取  消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
